rules/aip0133/aip0133.go

```diff
@@ -55,10 +55,10 @@ func isCreateRequestMessage(m *desc.MessageDescriptor) bool {
 // get resource message type name from method
 func getResourceMsgName(m *desc.MethodDescriptor) string {
 	if !isCreateMethod(m) {
-		return ""
+		return nil
```

Comment:
I think the return value can't be nil because the return type is string

Reply:
Honestly, you should be able to ignore this case -- you should only be calling this method once you already know that it is a create method.

Reply:
Make sense. I delete the check in both getResourceMsgName and getResourceMsgNameFromReq to simplify the logic.